src/platform-includes/session-replay/setup/javascript.mdx

```diff
@@ -46,3 +46,50 @@ Sentry.init({
 const { Replay } = await import("@sentry/browser");
 Sentry.addIntegration(new Replay());
 ```
+
+### Canvas Recording
+
+<Alert level="warning">
+  There is currently no PII scrubbing in canvas recordings!
+</Alert>
+
+In order to record HTML canvas elements, you will need to add an additional integration in your Sentry configuration. The canvas integration is exported from the browser SDK, so no additional package is required. Canvas recording is opt-in and will be tree-shaken from your bundle if it is not used:
+
+```js
+// import Sentry from your framework SDK (e.g. @sentry/react) instead of @sentry/browser
+import * as Sentry from "@sentry/browser";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+
+  integrations: [
+    new Sentry.Replay(),
+
+    // The following is all you need to enable canvas recording
+    new Sentry.ReplayCanvas(),
+  ],
+});
+```
+
+#### Canvas Recording Performance
```

Comment:
I'd move this out of the setup page if not everyone will encounter it. 

Maybe into troubleshooting is better, then from here we can link over to it.

Reply:
I'd prefer to keep the instructions out of troubleshooting, I think troubleshooting -> docs is better as folks should be aware of the perf implications when running canvas while setting it up. It could make sense to have this in configuration and point both troubleshooting + setup to the configuration page.

Reply:
fair

im down to keep it simple. I do think after the docs -> nextjs migration happens we should look at adding some more pages into our left sidebar for replay in order to make things easier to find. the right-sidebar of pages like troubleshooting are hard for me to parse.